packages/commonwealth/client/scripts/views/components/component_kit/new_designs/cw_button.tsx

```diff
@@ -62,7 +59,10 @@ export const CWButton = (props: ButtonProps) => {
   } = props;
 
   return (
-    <div className={`btn-border ${buttonType} ${disabled ? 'disabled' : ''}`}>
+    <div
+      className={`btn-border ${buttonType} ${disabled ? 'disabled' : ''}`}
+      {...widthStyle(buttonWidth)}
+    >
```

Comment:
1. we should avoid injecting inline style (it is harder to override them later on), proper solution would be here to add class similarly to `buttonType` and handle it in the css file
2. take a look on `[clsx](https://www.npmjs.com/package/clsx)` that is great for building the conditional classes. We use it [here](https://github.com/hicommonwealth/commonwealth/blob/c8ed464e9569bc938e4f4448a712df97ca817c2c/packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx#L220) for example

Reply:
Addressed.

At first I thought that adding that code on the .tsx file would be a better option because it would be less code added to the .css file, but indeed a better practice to avoid adding that on the React file.